test/desdemona/plugins/swift_test.clj

```diff
@@ -0,0 +1,152 @@
+(ns desdemona.plugins.swift-test
+  (:require [clojure.test :refer [deftest is]]
+            [cheshire.core :as json]
+            [camel-snake-kebab.core :refer [->kebab-case-keyword]]
+            [clojure.java.io :as io]
+            [clj-time.core :as t]
+            [clj-http.fake :as http]
+            [onyx.peer.pipeline-extensions :as p-ext]
+            [onyx.peer.function :as function]
+            [desdemona.plugins.swift :as swift])
+  (:import (org.joda.time DateTimeUtils)))
+
+(defmacro freeze-time [time & forms]
+  `(try
+     (DateTimeUtils/setCurrentMillisFixed (.getMillis ~time))
+     ~@forms
+     (finally
+       (DateTimeUtils/setCurrentMillisSystem))))
+
+(deftest get-cloud-files-test
+  (let [cloud-files {:name "cloudFiles" :type "object-store" :id 1}
+        auth-response {:access {:service-catalog [{:name "one"}
+                                                  {:name "two" :type "object-store"}
+                                                  {:name "cloudFiles" :type "something"}
+                                                  cloud-files
+                                                  {:type "object-store"}]}}
+        got (swift/get-cloud-files auth-response)]
+    (is (= cloud-files got))))
+
+(deftest get-token-test
+  (let [token {:id 1}
+        auth-response {:access {:token token}}
+        got (swift/get-token auth-response)]
+    (is (= token got))))
+
+(deftest authenticate-test
+  "This test verifies that the authenticate function makes a request to the
```

Comment:
@sirsean `lein eastwood` should stop complaining if you put your test descriptions inside of the `testing` macro.
```clojure
(deftest authenticate-test
  (testing "This test verifies..."
    (the-test-body)))
```
Here is the source for the testing macro: https://github.com/clojure/clojure/blob/d5708425995e8c83157ad49007ec2f8f43d8eac8/src/clj/clojure/test.clj#L596-L602